client/verta/verta/_internal_utils/http_session_utils.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you're up for it, would you mind renaming this file `http_session.py`? The `*_utils.py` scheme was kind of a mistake 😬